app/services/import_matches.rb

```diff
@@ -0,0 +1,41 @@
+class ImportMatches < Patterns::Service
+  
+  def call
+    [1, 2].each do |week|
```

Comment:
You need another spec that includes 3 weeks in test... as this is clearly wrong.